.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
+  
```

Comment:
- [ ] Kindly add the eslint for javascript so that the best practices can guide you through as I see this is your first JS project. You can find it [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)